### ISSUE TITLE
add feature to generate output without namespace

### ISSUE THREAD

**Issue by abenhamdine** [SweetIQ/schemats#25]
Currently, namespace parameter is required.

If one runs schemats without argument -n (or with empty one) like that : 
`schemats generate -c postgres://postgres@localhost/osm -t users -o osm.ts`

It ends up with the following output in osm.ts : 
```ts
export namespace undefined {

    export namespace usersFields {
...
```

Using a namespace for a whole module in typescript is ok, but a bit outdated.
With es2015 modules, it seems preferable to generate directly : 
```ts
    export namespace usersFields {
...
```

wich allows to import as following : 
```ts
import * as myNamespaceInConsumerModule from 'osm.ts'
...
```

It would be nice to make namespace optional.

**Comment by xiamx** [comment]
You are right. 


I'm think about deprecating the namespace parameter all together. The choice of having a namespace was due to my inexperience with Typescript earlier, and mixed information from Typescript 1.5 days. As of today, and using Typescript 2.0, there's really no motivation to keep using namespaces

**Comment by abenhamdine** [comment]
Totally agree :thumbsup:
As a consequence of semver, it will be schemats 1.0 no ?   💃 :tada:

**Comment by xiamx** [comment]
Fixed by #32 https://github.com/SweetIQ/schemats/commit/64143dd352ff1824417c288b248b016a003b4697
